Documentation/observability/metrics.rst

```diff
@@ -433,6 +433,8 @@ Name                                     Labels
 ``identity_gc_latency``                  ``outcome``, ``identity_type``                     Enabled    Duration of the last successful identity GC run
 ``ipcache_errors_total``                 ``type``, ``error``                                Enabled    Number of errors interacting with the ipcache
 ``ipcache_events_total``                 ``type``                                           Enabled    Number of events interacting with the ipcache
+``identity_cache_timer_duration``        ``name``                                           Enabled    Seconds required to execute periodic policy processes. ``name="id-alloc-update-policy-maps"`` is the time taken to apply incremental updates to the BPF policy maps.
+``identity_cache_timer_trigger_latency`` ``name``                                           Enabled    Seconds spent waiting for a prevous process to finish before starting the next round. ``name="id-alloc-update-policy-maps"`` is the time taken to apply incremental updates to the BPF policy maps.
```

Comment:
Why is `timer_trigger_folds` not documented?

Reply:
mostly because I don't think it's that interesting :-). I was debating whether or not to drop the metric to begin with.

But yes, I'll document it.

Reply:
Cool! I suspected as much. Please note that `TriggerFolds` seems unused at the moment